Polyhedron/demo/Polyhedron/Plugins/IO/CMakeLists.txt

```diff
@@ -76,9 +76,15 @@ endif()
 
 find_path(3MF_INCLUDE_DIR
   NAMES Model/COM/NMR_DLLInterfaces.h
+  PATHS /usr/include
+        /usr/local/include
   DOC "Path to lib3MF headers"
   )
-find_library(3MF_LIBRARIES NAMES 3MF DOC "Path to the lib3MF library")
+find_library(3MF_LIBRARIES
+  NAMES 3MF DOC
+  PATHS /usr/lib
+  /usr/local/lib
+  /usr/lib64"Path to the lib3MF library")
```

Comment:
This patch is wrong:
  - syntax error,
  - anyway you are added hints to places that already searched for by CMake.

https://cmake.org/cmake/help/v3.9/command/find_package.html

Reply:
@sloriot complained that they were missing, so I added the same as the ones in our FindIPE. If those are useless, then FindIPE has the same problem.